pilot/pkg/security/trustdomain/bundle_test.go

```diff
@@ -0,0 +1,104 @@
+// Copyright 2019 Istio Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package trustdomain
+
+import (
+	"reflect"
+	"testing"
+)
+
+func TestReplaceTrustDomainAliases(t *testing.T) {
+	type inStruct struct {
+		trustDomainBundle TrustDomainBundle
+		users             []string
+	}
+	testCases := []struct {
+		name   string
+		in     inStruct
```

Comment:
nit: I think you can just put both `trustDomainBundle`, `users` directly here, no need for the `inStruct`.

Reply:
Done. Thanks